internal/node/explorer.go

```diff
@@ -0,0 +1,91 @@
+package node
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+)
+
+var client = &http.Client{Timeout: 30 * time.Second}
+
+type (
+	Explorer interface {
```

Comment:
I would move `explorer.go` and `pinmanager.go` to `/internal/bus` since they are required by the bus. The `node` packages only purpose should really be acting as the glue between the different APIs. Not implement any requirements that packages might have.

Also we don't need the `Explorer` interface afaik since we already got the `ExchangeRateProvider` interface.

Reply:
`PinManager` shouldn’t be in internal since it’s needed to initialize a `Bus`

Reply:
Oh yeah if it's in `bus.New` we need it out of `internal`.